bin/MadGraph5_aMCatNLO/gridpack_generation.sh

```diff
@@ -119,8 +119,8 @@ if [ ! -d ${AFS_GEN_FOLDER}/${name}_gridpack ]; then
 #   export SCRAM_ARCH=slc6_amd64_gcc472 #Here one should select the correct architechture corresponding with the CMSSW release
 #   export RELEASE=CMSSW_5_3_32_patch3
 
-  export SCRAM_ARCH=slc6_amd64_gcc481
-  export RELEASE=CMSSW_7_1_23
+  export SCRAM_ARCH=slc6_amd64_gcc530
```

Comment:
ah, so is intended that we switch to gcc5 also for MG5_aMC@NLO. This is an even stronger case to urgently discuss with computing how to handle the production of MC events for 2016 data analysis in the future...